cob_omni_drive_controller/src/control_plugin.cpp

```diff
@@ -81,6 +94,23 @@ class WheelController: public GeomController<hardware_interface::VelocityJointIn
             steer_joints_[i].setCommand(wheel_commands_[i].dVelGearSteerRadS);
             drive_joints_[i].setCommand(wheel_commands_[i].dVelGearDriveRadS);
         }
+        
+        if(cycles_ < pub_divider_ && (++cycles_) == pub_divider_){
+            if(commands_pub_->trylock()){
+                ++(commands_pub_->msg_.header.seq);
```

Comment:
This sequence number is part of the Header  entry, I don't see any code for handling the number in realtime publisher.